### PR TITLE
Introduce e2e environments for AMPHTML ads

### PR DESCRIPTION
**Pull request by lannka**

### CODE REVIEW

**Comment by lannka** [comment]
@estherkim do you know what's a typical cause of this timeout error:

> TimeoutError: Waiting for element to be located amp-ad > iframe

https://travis-ci.org/ampproject/amphtml/jobs/569574768#L3824

**Comment by estherkim** [comment]
Hmm why is it timing out after ~5s? We set the timeout for `beforeEach` to 30s.
https://github.com/ampproject/amphtml/blob/e6839ffd827990ddf2f647dce4d71af7cf1e2808/build-system/tasks/e2e/describes-e2e.js#L352-L357
Can you try removing the inner `describe()` so that there are only `it()`s inside `describes.endtoend()`?

File: test/e2e/test-amp-bind-video.js [hunk 1]
```diff
@@ -15,9 +15,10 @@
  */
 
 describes.endtoend(
-  'amp-bind',
+  'amp-bind with <amp-video>',
   {
     testUrl: 'http://localhost:8000/test/fixtures/e2e/amp-bind/bind-video.html',
+    environments: 'ampdoc-amp4ads-preset',
```
Comment (estherkim) [inline test/e2e/test-amp-bind-video.js]: This env isn't defined? Also should be an array i.e. `['a4a-fie']`

Comment (lannka) [reply test/e2e/test-amp-bind-video.js]: I introduced a concept of preset, which represent a collection of envs. Let me know if it works for you

Comment (estherkim) [reply test/e2e/test-amp-bind-video.js]: oh I see, looks good

File: build-system/server-inabox-template.html [hunk 2]
```diff
@@ -22,6 +22,8 @@ <h1>In-a-box Envelope</h1>
   <div style="background: #eee; overflow: hidden;; height: OFFSET;">scroll down to see the ad</div>
 
   <iframe
+      id="inabox-frame"
+      src="AD_URL"
```
Comment (zombifier) [inline build-system/server-inabox-template.html]: Duplicate `src` attribute (see below), which might cause errors. I think this line snuck in during the merge.

Comment (lannka) [reply build-system/server-inabox-template.html]: good catch!

**Comment by lannka** [comment]
@estherkim @powerivq PTAL

**Pull request merged**